blueflood-elasticsearch/src/main/resources/init-es.sh

```diff
@@ -4,13 +4,25 @@
 #  are required to properly tokenize the incoming data.
 # You can pass it a url if your elastic search doesn't reside on localhost:9092
 #WARNING: this script will destroy existing ES data and reset it to the proper init state
-
+# Example:
+#    Local ES: ./init-es.sh
+#    ES on OR: ./init-es.sh <url for OR:ES> <username> <password>
 ELASTICSEARCH_URL=${1:-"http://localhost:9200"}
+ES_USERNAME=${2:-}
+ES_PASSWD=${3:-}
```

Comment:
Having 3 command line params, should we should be using getopts or another command line parser?

http://wiki.bash-hackers.org/howto/getopts_tutorial